wiki/Tournaments/GTS/AGTS_2019/fr.md

```diff
@@ -3,7 +3,6 @@ tags:
   - AGTS 2019
   - AGTS
   - GTS
-outdated_translation: true
```

Comment:
This should be done in a separate PR.

Reply:
Rhaaaaaa, I didn't change my branch, correcting this real quick.

Reply:
Okay, done now.

Reply:
Sorry if I expressed myself poorly but the change was meant to be reverted. In other words, readd the line, so that the file will not show up in `git diff`.

In case you are unsure of what to do, just wait for a maintainer to pick this up and they will take care of it.

Reply:
Yeah, I'm not that sure about what I'm doing, so I won't try, risking to break the commit lol. Thanks for taking time for me, I'm kinda clumsy.